postgres/locations.go

```diff
@@ -50,16 +50,30 @@ func (ls *LocationsStore) Create(l *photohistory.Location) (*photohistory.Locati
 
 // Update a location
 func (ls *LocationsStore) Update(l *photohistory.Location) error {
+	q := `UPDATE locations
+			SET (id, alias, name, latitude, longitude, elevation)
+			VALUE (UUID_TO_BIN(UUID()), :alias, :name, :latitude, :longitude, :elevation)`
+	_, err := ls.db.NamedExec(q, l)
```

Comment:
I referenced your implementation of `Create()`. I have never seen `UUID_TO_BIN` before. is this a function supported by Postgres?

I also came across this example, which is similar:

	_, err = db.NamedExec(`UPDATE user SET first_name=:first, last_name=:last WHERE first_name = 'Bin'`,
		map[string]interface{}{
			"first": "Ben",
			"last":  "Smith",
			"email": "bensmith@allblacks.nz",
		})

Could a struct(Location) be used in substitute of a map (shown above)?

Reply:
Yes, UUID_TO_BIN is an Postrest function.  It generates a UUID.  Alternately we could generate the UUID vi Go and pass it in.

In the Update function we don't need to generate a UUID because the Location will already have one.  We just need to update the other fields.

Reply:
Ok, I will make these changes and begin implementing 'users' and 'photos' later today